lib/indexer/indexer.js

```diff
@@ -470,6 +470,19 @@ class Indexer extends EventEmitter {
     ;
   }
 
+  /**
+   * Prune block indexing
+   * Indexers will implement this method to prune indexing for the block
```

Comment:
The idea is that this method is empty so that it always will retain the filters even if the block data is pruned right?

Reply:
It's unimplemented.

Reply:
Just making sure that it doesn't need to be implemented. bip 157 mentions:

```
Nodes MAY prune block data after generating and storing all filters for a block.
```

https://github.com/bitcoin/bips/blob/master/bip-0157.mediawiki#node-operation

Reply:
That isn't related to this. This is related to _removing_ a filter, when it's removed.

Reply:
It may help to look at the context of the code here, this is for the base indexer.

Reply:
You are correct, I was thinking it was on the `FilterIndexer`